src/components/text-editor/prosemirror-adapter/plugins/table-plugin.ts

```diff
@@ -0,0 +1,42 @@
+import { tableNodes, tableEditing } from 'prosemirror-tables';
+import { Plugin } from 'prosemirror-state';
+
+export const getTableEditingPlugins = (tablesEnabled: boolean): Plugin[] => {
+    if (tablesEnabled) {
+        return [tableEditing()];
+    }
+
+    return [];
+};
+
+export const getTableNodes = () => {
+    return tableNodes({
+        tableGroup: 'block',
+        cellContent: 'block+',
+        cellAttributes: {
+            background: {
+                default: null,
+                getFromDOM: (dom) => {
+                    return dom.style.backgroundColor || null;
+                },
+                setDOMAttr: (value: string, attrs: Record<string, string>) => {
+                    if (value) {
+                        attrs.style =
+                            (attrs.style || '') + `background-color: ${value};`;
```

Comment:
```suggestion
                            (attrs.style || '') + ` background-color: ${value};`;
```

I think we need to add a space, like that ☝️, right? 🤔

Reply:
Ah, oh… no, of course, as long as the existing value ends with a semi-colon, we don't need the space 🤦‍♂️